test/integration-test.sh

```diff
@@ -42,4 +44,6 @@ case $1 in
 esac
 
 cd "$DIR"
-STUPID_LOGLEVEL=debug ./stupid $CONFIG_FILE 100
+# STUPID_LOGLEVEL=debug ./stupid $CONFIG_FILE 100
+# docker run
```

Comment:
if we don't need them, pls kill these lines. otherwise, pls comment on why we need to keep them